[rospy.client][INFO] 2020-09-02 04:09:50,008: init_node, name[/rostopic_2601_1599019789959], pid[2601]
[xmlrpc][INFO] 2020-09-02 04:09:50,009: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2020-09-02 04:09:50,010: Started XML-RPC server [http://192.168.137.54:34359/]
[rospy.init][INFO] 2020-09-02 04:09:50,011: ROS Slave URI: [http://192.168.137.54:34359/]
[rospy.impl.masterslave][INFO] 2020-09-02 04:09:50,011: _ready: http://192.168.137.54:34359/
[rospy.registration][INFO] 2020-09-02 04:09:50,012: Registering with master node http://192.168.137.54:11311
[xmlrpc][INFO] 2020-09-02 04:09:50,012: xml rpc node: starting XML-RPC server
[rospy.init][INFO] 2020-09-02 04:09:50,112: registered with master
[rospy.rosout][INFO] 2020-09-02 04:09:50,113: initializing /rosout core topic
[rospy.rosout][INFO] 2020-09-02 04:09:50,118: connected to core topic /rosout
[rospy.simtime][INFO] 2020-09-02 04:09:50,121: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rospy.internal][INFO] 2020-09-02 04:09:50,411: topic[/camera/rgb/image_raw] adding connection to [http://192.168.137.54:37013/], count 0
[rospy.msg][ERROR] 2020-09-02 04:09:50,432: cannot deserialize message: EXCEPTION Traceback (most recent call last):
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/msg.py", line 223, in deserialize_messages
    msg_queue.append(data.deserialize(q))
  File "/opt/ros/melodic/lib/python2.7/dist-packages/sensor_msgs/msg/_Image.py", line 147, in deserialize
    codecs.lookup_error("rosmsg").msg_type = self._type
LookupError: unknown error handler name 'rosmsg'

[rospy.internal][ERROR] 2020-09-02 04:09:50,434: Traceback (most recent call last):
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 735, in receive_once
    p.read_messages(b, msg_queue, sock)
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 398, in read_messages
    deserialize_messages(b, msg_queue, self.recv_data_class, queue_size=self.queue_size)
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/msg.py", line 245, in deserialize_messages
    raise genpy.DeserializationError("cannot deserialize: %s"%str(e))
DeserializationError: cannot deserialize: unknown error handler name 'rosmsg'

[rospy.internal][WARNING] 2020-09-02 04:09:50,439: Unknown error initiating TCP/IP socket to 192.168.137.54:40247 (http://192.168.137.54:37013/): Traceback (most recent call last):
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 562, in connect
    self.read_header()
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 657, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rosgraph/network.py", line 378, in read_ros_handshake_header
    return decode_ros_handshake_header(bval)
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rosgraph/network.py", line 339, in decode_ros_handshake_header
    raise ROSHandshakeException("Invalid line in handshake header: [%s]"%line)
ROSHandshakeException: Invalid line in handshake header: []

[rospy.internal][INFO] 2020-09-02 04:09:50,440: topic[/camera/rgb/image_raw] removing connection to http://192.168.137.54:37013/
[rospy.internal][INFO] 2020-09-02 04:09:50,460: topic[/rosout] adding connection to [/rosout], count 0
[rospy.internal][INFO] 2020-09-02 04:09:53,410: topic[/rosout] removing connection to /rosout
[rospy.core][INFO] 2020-09-02 04:09:57,259: signal_shutdown [signal-2]
[rospy.registration][WARNING] 2020-09-02 04:09:57,263: cannot unregister with master due to network issues
[rospy.impl.masterslave][INFO] 2020-09-02 04:09:57,264: signal-2
[rospy.core][INFO] 2020-09-02 04:09:57,381: signal_shutdown [atexit]
